deepchem/models/torch_models/graphrnn.py

```diff
@@ -0,0 +1,309 @@
+import torch
+import torch.nn as nn
+from deepchem.models.torch_models import TorchModel
+from deepchem.models.losses import L2Loss, SoftmaxCrossEntropy
+
+
+class NodeFeatureMLP(nn.Module):
```

Comment:
Can you please add this class and `EdgeRNN` to `layers.py` in a separate PR to facilitate reviewing?